### PR TITLE
[66] Make it possible to hide strapline

### PR DESCRIPTION
**Pull request by tm-kn**
Relies on torchbox/wagtail-torchbox#163 being merged to the backend
codebase.
https://projects.torchbox.com/projects/tbxcom-redesign/tickets/66

### CODE REVIEW

File: src/templates/culture/culture-page.module.scss [hunk 1]
```diff
@@ -38,6 +38,10 @@
                 line-height: 100px;
                 margin: 0 8.33vw 20px 16.66vw;
             }
+
+            &Hidden {
+                display: none;
```
Comment (helenb) [inline src/templates/culture/culture-page.module.scss]: This needs to be an accessibly visually hidden - `display:none` will hide it from screen readers which is not what we want. Any recent wagtail build should have a `visually-hidden` mixin you can nick.

**Comment by helenb** [comment]
Needs the way the text is hidden to be fixed before this can be merged.

File: src/templates/culture/culture-page.module.scss (hunk 1 above)
Comment (tm-kn) [reply src/templates/culture/culture-page.module.scss]: Updated.

**Comment by helenb** [comment]
Update looks good. Happy for you to merge when you are ready, @tm-kn

**Pull request merged**